include/gz/sim/MeshInertiaCalculator.hh

```diff
@@ -0,0 +1,230 @@
+/*
+ * Copyright (C) 2019 Open Source Robotics Foundation
```

Comment:
```suggestion
 * Copyright (C) 2023 Open Source Robotics Foundation
```

Reply:
Corrected in [ee6f863](https://github.com/gazebosim/gz-sim/pull/2061/commits/ee6f863535a4f9fabd39e914f9bf5861f5d939da)